admin/config.js

```diff
@@ -16,5 +16,6 @@ module.exports = {
   ROLE_TEST_DEVELOPER: 'TEST-DEVELOPER',
   ROLE_TEACHER: 'TEACHER',
   HOME_TEST_DEVELOPER: '/administrator',
-  HOME_TEACHER: '/school/school-home'
+  HOME_TEACHER: '/school/school-home',
```

Comment:
we don't want routes in the config file.  I asked Paula to make a similar change in another PR, please ask her about what she did to change this.  Think she made a routeConfig.js file?

Reply:
@GeorgeChatzigiannis - looks like these routes are still here.

Reply:
@GuyHarwood @jon-shipley I'll have to ask @pris54 about them since they were introduced from her on this commit bcf8766dcc3399f7e503cb93c4e9b4b5c001f210

Reply:
👍🏻 Please